sf-docker/jetson-docker-compose.yml

```diff
@@ -92,6 +92,7 @@ services:
       - Database__DbEngine
       - AppSettings__Log-RollingFile-Enabled
       - S3Bucket__Endpoint
+      - GstPipelineTemplate
```

Comment:
IMO the Nx camera is the only one that should not need the pipeline as it should consume grpc source

Reply:
removed